library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java

```diff
@@ -51,7 +50,7 @@ public static String getDeletedOrMentionedText(Message message) {
                 text = text.replace("@" + userName, "**" + "@" + userName + "**");
             }
         }
-        return text;
+        return text.replaceAll("\n", "<br/>  <br/>  \n");
```

Comment:
This shouldn't work. Have a look at the tags we allow on JS. HTML should not be allowed. Its a security issue

![image](https://user-images.githubusercontent.com/265409/66526768-fd87e580-eab6-11e9-8334-962dc0748bcf.png)